### ISSUE TITLE
Offsets wrong in other languages

### ISSUE THREAD

**Issue by atiksoftware** [w1nds/pubg-mobile-esp#3]
hi,
i am using Turkish language version.
when search for cars, it find peoples as a car like in picture.
cant found cars, cant founds weapons ...

i wanna ask, whats mind of search mothods for points ?.
how did you find the points ?
i think chinesee GameData::bGoodsTag different from turkish version.
how can i find ?

Good to know:
i add a some code to main function, for detect FullScreenMode and resize resolation for play in full screen.
if you interest , i can share with you.

and i see a little problem in search in memory, its using high cpu,
it can be more performance.

and i will add sprite icons for weapons and cars to directx drawing,
i think it will be well.

but i have problem on find true points :)
actualy, i never work on game hacking before, its the first.

thank you for interest

(i write random text with latin chars on menu bla bla because of i get wrong from visual studio)
![Screenshot_6](https://user-images.githubusercontent.com/14994715/54279673-baa87280-45a6-11e9-9eeb-8bb7cb6089bb.jpg)

**Comment by atiksoftware** [comment]
喜
我使用土耳其語版本。
在搜索汽車時，它會發現人們就像汽車一樣。
無法找到汽車，無法找到武器......

我想問一下，搜索方法的重點是什麼？
你是怎麼找到積分的？
我認為中文GameData :: bGoodsTag與土耳其版不同。
我怎麼能找到？

很高興知道：
我為main函數添加了一些代碼，用於檢測FullScreenMode並調整resolation以便全屏播放。
如果您有興趣，我可以與您分享。

我在內存中看到一個小問題，它使用高CPU，
它可以更多的表現。

我將為武器和汽車添加精靈圖標到directx繪圖，
我認為會很好。

但我找到真正的問題:)
實際上，我從來沒有在遊戲黑客工作，它是第一個。

謝謝你的興趣

（我在菜單bla bla上用拉丁字符寫隨機文本因為我從visual studio出錯了）

**Comment by w1nds** [comment]
The turkish version should be the same as the international version. You can refer to the content on [unknowncheats](https://www.unknowncheats.me/forum/playerunknown-s-battlegrounds/).

**Comment by OprekerSejati** [comment]
what version visual studio you recomended opening your solution?

**Comment by atiksoftware** [comment]
i used visual studio 2017.
i get much error first time, then i create files(.h and .cpp) one by one.
its compiled. 
But the problem is: if you using international version, its not work. search bytes not equal. 
ex: you will find humans when searching cars.
But if you have idea of ofsets, i can help to you.


(maybe you already known)
But you should set **include** and **lib** directory to project properties.
and you should select true directx versions for debug/release and x64/x86

**Comment by OprekerSejati** [comment]
@atiksoftware  thank you very much for your direction mate

**Comment by atiksoftware** [comment]
yes, turkish version same to international. i searched unknowncheats for all posts.

but i think, its not good method as search bytes in process.
its make realy high cpu usage (70%+)
i think, should be points and ofsets for entity count and and per entity.
if we look to points, I believe its its will work more performance.

**Comment by OprekerSejati** [comment]
@atiksoftware 

Yeah, the problem is how to find Uworld, so it will easy and less cpu usage

**Comment by atiksoftware** [comment]
@OprekerSejati yea, you are right, 
actualy I have no experience about game hacking.

i did create as library for draw esp(bones, weapon icons ...) with game engine, and its working with 0.3% cpu usage.

but i locked on ofsets

**Comment by atiksoftware** [comment]
@OprekerSejati i have not idea about "access UE4 api directly"

**Comment by OprekerSejati** [comment]
@atiksoftware actually not "directly" , in UC there are ton of old pubg pc ESP source code , it will be easier to read it for understanding

**Comment by OprekerSejati** [comment]
@atiksoftware do you mind share your working with 0.3% cpu source code? i will glad if we can continue this on discord

**Comment by atiksoftware** [comment]
@OprekerSejati 
you can see demo drawer with random data
https://drive.google.com/file/d/1xZuRE2e9ROkAhTo_C7ZNj_Zqr7JPtvBG/view?usp=sharing

**Comment by atiksoftware** [comment]
@OprekerSejati ok, can you give me code of your discord account

**Comment by OprekerSejati** [comment]
@atiksoftware please add my discord Kal-El#7402

**Comment by atiksoftware** [comment]
@OprekerSejati i add you

**Comment by OprekerSejati** [comment]
@atiksoftware yess, I already send message on discord

**Comment by VonalOrdu** [comment]
> i used visual studio 2017.
> i get much error first time, then i create files(.h and .cpp) one by one.
> its compiled.
> But the problem is: if you using international version, its not work. search bytes not equal.
> ex: you will find humans when searching cars.
> But if you have idea of ofsets, i can help to you.
> 
> (maybe you already known)
> But you should set **include** and **lib** directory to project properties.
> and you should select true directx versions for debug/release and x64/x86

merhaba derlemeyi bir türlü başaramadım bu konuda bana yardımcı olurmusun?
DirectX versiyonu ne kullanıyorsun?
Proje dosyalarını paylaşırmısın?
Bende offsetleri bulma konusunda sana yardımcı olmaya çalışırım.

**Comment by atiksoftware** [comment]
@VonalOrdu Merhaba dostum,
Ben de derleme konusunda sıkıntı yaşadım, saç sapan hatalar vardı (sanırım proje dosyası başka bilgisayardan geldiği için), sonra yeni proje açıp büyün dosyaları tek tek oluşturdum. Header ve cpp dosyalarını.

Sonrasında directx ile ilgili bi sorun yaşadım.
sonra direcx SDK kurdum
sanrıım linki : https://www.microsoft.com/en-us/download/details.aspx?id=6812
emin değilim, 3-5 farklı şey denedim.
directx9 kurum ama

sonra asıl nokta
**Project** > **Properties** dan
**VC++ Directory** kısmına gel
**Include Directory** e "C:\Program Files (x86)\Microsoft DirectX SDK (March 2009)\Include"
şeklinde directx sdk yı nereye kurduysan onun içindeli Include klasörünün adresini ekle
aynı şekilde Library Directory için SDK nin içindeki Lib klaösrünün adresini ver.
Örn: 
"C:\Program Files (x86)\Microsoft DirectX SDK (March 2009)\Include"
"C:\Program Files (x86)\Microsoft DirectX SDK (March 2009)\Lib\x86"
VStudio directx için include edeceği ve kütüphanelerin dosyalarına ihtiyacı var 
projeyi x86 modda derledim. 

(x64 te çalışınca process list te x86 uygulamaları göremedim. aow_exe.exe de x86 olarak çalışıyor.)
elbet bir çözümü vardır, ama şuanlık bu konuda bilgim yok

Resimdeki gibi..
![Screenshot_4](https://user-images.githubusercontent.com/14994715/54857635-79843100-4d11-11e9-8d32-3dc09bb1683a.jpg)

**Comment by atiksoftware** [comment]
@VonalOrdu ve sanırım directx te bazı fonsiyonlar debug imkanı sunmuyor, bu sebeple debug modda açmak istediğinde hata verebiliyor.
(o sıra SFML üzerinde de çalışıyodum, onda da olabilir bu sorun. sadece aklında bulunsun diye dedim)

**Comment by atiksoftware** [comment]
@VonalOrdu Ofset bulma konusunda deneyimim yok öncesinde,
Bu projede ofset üstünden gitmemiş, ByteArray araması ile ilerliyor , bu da işlemcinin irzına geçiyor.
ofset bulma konusunda bilgin varsa güzel şeyler çıkartabiliriz

**Comment by VonalOrdu** [comment]
> @VonalOrdu Merhaba dostum,
> Ben de derleme konusunda sıkıntı yaşadım, saç sapan hatalar vardı (sanırım proje dosyası başka bilgisayardan geldiği için), sonra yeni proje açıp büyün dosyaları tek tek oluşturdum. Header ve cpp dosyalarını.
> 
> Sonrasında directx ile ilgili bi sorun yaşadım.
> sonra direcx SDK kurdum
> sanrıım linki : https://www.microsoft.com/en-us/download/details.aspx?id=6812
> emin değilim, 3-5 farklı şey denedim.
> directx9 kurum ama
> 
> sonra asıl nokta
> **Project** > **Properties** dan
> **VC++ Directory** kısmına gel
> **Include Directory** e "C:\Program Files (x86)\Microsoft DirectX SDK (March 2009)\Include"
> şeklinde directx sdk yı nereye kurduysan onun içindeli Include klasörünün adresini ekle
> aynı şekilde Library Directory için SDK nin içindeki Lib klaösrünün adresini ver.
> Örn:
> "C:\Program Files (x86)\Microsoft DirectX SDK (March 2009)\Include"
> "C:\Program Files (x86)\Microsoft DirectX SDK (March 2009)\Lib\x86"
> VStudio directx için include edeceği ve kütüphanelerin dosyalarına ihtiyacı var
> projeyi x86 modda derledim.
> 
> (x64 te çalışınca process list te x86 uygulamaları göremedim. aow_exe.exe de x86 olarak çalışıyor.)
> elbet bir çözümü vardır, ama şuanlık bu konuda bilgim yok
> 
> Resimdeki gibi..
> ![Screenshot_4](https://user-images.githubusercontent.com/14994715/54857635-79843100-4d11-11e9-8d32-3dc09bb1683a.jpg)

Cevapların için teşekkürler derlemeyi başardım sayenizde.
Lakin bu seferde aşağıdaki kısımda hata veriyor bilginiz varmı ?3
`	if (!bRet || hDevice == NULL)
	//	if (!bRet )
	{
		printf("baslatmabasarizoldu!\n");
		system("pause");
		return 1;
	}`

**Comment by atiksoftware** [comment]
@VonalOrdu aah, :)
sanırım servis çalıştırma satırı orası.
o kısımdaki if i sil.
Onun amacı şu : Pubg için yapılmış hilelerin hepsinde var sanırım, oyun içinde bir değeri değiştirmek yada aimhack gibi mudehalede bulunmak için WriteMemory yetkisiz bir fonksiyon. başarısız oluyor.
Bu sebeple bir SYS dosyası üzerinden bir hizmet/servis çalıştırıyorlar, o hizmet aracılığı ile aow_exe.exe ye memorywrite yaptırıyorlar. 
bu konuda ileri seviye bilgim yok, fakat gördüğüm kadarı ile o sys dosyası memorywrite için aracı bir api gibi bişi.
esp için buna gerek yok, read memory yeterli.
o yüzden o satırları silebilirsin.

**Comment by letiendunggithub** [comment]
I have built the source code and had many problem :3

**Comment by letiendunggithub** [comment]
can you share your source ? atiksoftware

**Comment by atiksoftware** [comment]
@letiendunggithub Whats your problems
